omahelsinki/templates/base.html

```diff
@@ -67,9 +67,9 @@
                 {{ LANGUAGE_CODE|upper }}
               </a>
               <div class="dropdown-menu dropdown-menu-right" aria-labelledby="navbarDropdown">
-                <a class="dropdown-item{% if LANGUAGE_CODE == 'fi' %} active{% endif %}" href="#" data-language="fi">suomi</a>
+                <a class="dropdown-item{% if LANGUAGE_CODE == 'fi' %} active{% endif %}" href="#" data-language="fi">Suomi</a>
+                <a class="dropdown-item{% if LANGUAGE_CODE == 'sv' %} active{% endif %}" data-language="sv" href="#">Svenska</a>
```

Comment:
Minor nitpick, could we have the same order for href="#" data-language attributes? Looks nicer :)

Reply:
fixed :)